src/components/Proposals/ProposalCard.js

```diff
@@ -1,23 +1,83 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Card, GU } from '@aragon/ui'
+import { Card, GU, textStyle, useTheme } from '@aragon/ui'
+import { DISPUTABLE_VOTE_STATUSES } from './disputable-vote-statuses'
+import ProposalOption from './ProposalOption'
+import DisputableStatusLabel from './DisputableStatusLabel'
 
 function ProposalCard({ vote, onProposalClick }) {
-  const { voteId, context } = vote
+  const theme = useTheme()
+  const { context, voteId } = vote
+
   return (
     <Card
       onClick={() => onProposalClick(voteId)}
       css={`
+        display: grid;
+        grid-template-columns: 100%;
+        grid-template-rows: 1fr auto auto;
+        grid-gap: ${1 * GU}px;
         padding: ${3 * GU}px;
       `}
     >
-      <p>{context}</p>
+      <div
+        css={`
+          // overflow-wrap:anywhere and hyphens:auto are not supported yet by
+          // the latest versions of Safari (as of June 2020), which
+          // is why word-break:break-word has been added here.
+          hyphens: auto;
+          overflow-wrap: anywhere;
+          word-break: break-word;
+        `}
+      >
+        <span css="font-weight: bold">#{voteId}: </span>
+        <span>{context || 'No description provided'}</span>
+      </div>
+      <ProposalOption
+        color={theme.positive}
+        percentage={(vote.yeas * 100) / vote.votingPower}
+        label={
+          <WrapProposalOption>
+            <span>Yes</span>
+          </WrapProposalOption>
+        }
+      />
+      <ProposalOption
+        color={theme.negative}
+        percentage={(vote.nays * 100) / vote.votingPower}
+        label={
+          <WrapProposalOption>
+            <span>No</span>
+          </WrapProposalOption>
+        }
```

Comment:
Do we anticipate needing to supply other more complex labels? If not it might be easier to nest `WrapProposalOption` styles within the component and pass less in the prop.